react/components/molecules/chat-message/chat-message.js

```diff
@@ -57,9 +59,17 @@ export class ChatMessage extends PureComponent {
         return { marginTop: index > 0 ? 2 : 0 };
     };
 
+    _chatMessageComponent = () => {
+        return this.props.onPress ? Touchable : View;

```

Comment:
Is it the whole message that is supposed to be touchable? If so, perhaps use `underlayColor` for iOS